scripts/scripts.js

```diff
@@ -14,8 +14,9 @@
  * log RUM if part of the sample.
  * @param {string} checkpoint identifies the checkpoint in funnel
  * @param {Object} data additional data for RUM sample
+ * @param {Object} extraData optional data for analytics tracking
  */
-export function sampleRUM(checkpoint, data = {}) {
+export function sampleRUM(checkpoint, data = {}, extraData = {}) {
```

Comment:
I implemented it this way because the full `data` object is converted to a json string and sent as the beacon body.
As the extra data is only needed for analytics and not for RUM, I extracted it to a separate object.
The alternative, would be to add to the beacon only the `data` properties that are relevant for RUM, i.e. `source, target`

Reply:
Finally replaced `extraData` by the property `element` in `data`